modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -184,11 +183,17 @@ public ObjectField addOrUpdateCustomObjectField(
 				state, objectFieldSettings);
 		}
 
+		String dbTableName = existingObjectField.getDBTableName();
+
+		if (Validator.isNull(dbTableName)) {
+			dbTableName = objectDefinition.getDBTableName();
```

Comment:
@seltong can we call the `objectFieldLocalService.updateDBTableName(existingObjectField.getObjectFieldId())` here?

Reply:
I believe we don't need this logic here. I will just use the `existingObjectField.getDBTableName()` to update the object field. What do you think?